components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/BrowserToolbar.kt

```diff
@@ -468,6 +468,12 @@ class BrowserToolbar @JvmOverloads constructor(
         }
     }
 
+    internal fun onEditCancelled() {
+        if (editToolbar.editListener?.onCancelEditing() == true) {
+            displayMode()
+        }
+    }
```

Comment:
Okay, really the last nit... ;)

Should this be:

`if (editToolbar.editListener?.onCancelEditing() != false) {`

Reasoning: We want to switch to display mode if the listener returns `true` OR if there's no listener. I guess now that would only happen if there's a listener and it returns `true`?

Reply:
Fixed :) I was wondering why the one above was doing `!= false` as that just seemed more confusing for no reason. Thanks for the clarification!